client/src/App.css

```diff
@@ -1,3 +1,3 @@
 body {
-    background: #e9967a;
-}
+  background: #e9967a;
```

Comment:
After using sass compiler you should not any .css file in the repo, only `.scss`